website/docs/developer/api/api-gateway.md

```diff
@@ -54,40 +66,56 @@ Catalog reporting API output example:
+| -- collectionId           | `str`       | The unique ID of collection to compare.                                                             |
+| -- id                     | `str`       | The unique ID of the granule.                                                                       |
+| -- createdAt              | `int`       | The time, in milliseconds since 1 January 1970 UTC, data was originally ingested into cumulus.      |
+| -- executionId            | `str`       | Step function execution ID from AWS.                                                                |
+| -- ingestDate             | `int`       | The time, in milliseconds since 1 January 1970 UTC, that the data was originally ingested into ORCA.|
+| -- lastUpdate             | `int`       | The time, in milliseconds since 1 January 1970 UTC, that information was updated.                   |
+| -- files                  | `Array[Object]`| Description and status of the files within the given granule.                                    |
+| -- name                   | `str`       | The name and extension of the file.                                                                 |
+| -- cumulusArchiveLocation | `str`       | Cumulus bucket the file resides in.                                                                 |
+| -- orcaArchiveLocation    | `str`       | ORCA S3 Glacier bucket the file resides in.                                                         |
+| -- keyPath                | `str`       | S3 path to the file including the file name and extension, but not the bucket.                      |
+| -- sizeBytes              | `str`       | Size in bytes of the file. From Cumulus ingest.                                                     |
+| -- hash                   | `str`       | Checksum hash of the file provided by Cumulus.                                                      |
+| -- hashType               | `str`       | Hash type used to calculate the hash value of the file.                                             |
+| -- version                | `str`       | AWS provided version of the file.                                                                   |
+
+
+The API returns status code 200 on success, 400 if `pageIndex` or  `endTimestamp` is missing and 500 if an error occurs when querying the database.
 
 ## Recovery granules API
 
-The `recovery/granules` API call returns detailed status of the granule.
+The `recovery/granules` API call relates to an ORCA recovery job status and returns detailed status of the granule.
 
 Recovery granules API input invoke URL example: `https://example.execute-api.us-west-2.amazonaws.com/orca/recovery/granules`
 
-#### Required input fields
-- granule_id(str)- The unique ID of the granule to retrieve status for.
+### Catalog reporting API input
+An example of the API input body is shown below:
+```json
+{
+  "granule_id": "6c8d0c8b-4f9a-4d87-ab7c-480b185a0250",
```

Comment:
May want to make granule_id value more realistic like `MOD14A1.061.H5V12.2020312.141531789` for example

Reply:
changed